Logger/LogFactory.cs

```diff
@@ -1,11 +1,69 @@
-﻿namespace Logger
-{
-    public class LogFactory
-    {
-        public BaseLogger CreateLogger(string className)
-        {
-            
-            return null;
-        }
-    }
-}
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class LogFactory
+    {
+        private string? FilePath;
+
+
+        private string? _ClassName;
+        public string ClassName
+        {
+            get
+            {
+                return _ClassName!;
+            }
+            set => _ClassName = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+
+
+        public BaseLogger CreateLogger(string className)

```

Comment:
```suggestion
        public BaseLogger? CreateLogger(string className)

```
This should be nullable, as you should be returning a null value if the file path is not configured.